lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission/accuracyAppellantsSubmission.step.js

```diff
@@ -12,19 +12,24 @@ const inaccuracyReasonInputId = 'inaccuracy-reason';
 const accurateSubmissionLabelId = 'accurate-submission-label';
 const sectionName = 'About the appeal';
 
```

Comment:
----
pretty much ever step-def file needed changing to work with "inject some data, go to the site.." approach
this involved a bunch of hacking into existing features where things like "Given X And Y" started causing chaos and making us load things twice etc.